tests/csm_read_configmap.py

```diff
@@ -75,7 +75,8 @@ def __call__(self):
 
 def main():
     fields = {'name': {'required': True, "type": 'str'},
-              'namespace': {'required': False, "type": "str", 'default': 'services'}}
+              'namespace': {'required': False, "type": "str", 'default': 'services'},
+              'key': {'required': False, "type": "str", 'default': 'foo'}}
```

Comment:
Is this key: foo line needed?

Reply:
Yes, it's a change in CSM 1.2. Without this change, the "real" csm_read_configmap.py fails on a missing key parameter.